batch-setup/batch_setup.py

```diff
@@ -148,34 +148,6 @@ def batch_setup(region_name, run_id, vpc_id, securityGroupIds, computeEnvironmen
         boto_iam, boto_ec2, instanceRoleName, instanceRoleName)
     print("Using ECS instance profile %s" % instanceProfileArn)
 
-    # Create the spot fleet role
-    # https://docs.aws.amazon.com/batch/latest/userguide/spot_fleet_IAM_role.html
-    spotIamFleetRoleName = "AmazonEC2SpotFleetRole"
```

Comment:
It would be neat if we could think of a way to keep support for spot instance types for folks where Spot is cheaper than on-demand. On the other hand, it might be enough to point to this commit and say "undo this one".

Reply:
we can certainly do it, it's just some plumbing